doc/source/user_guide/meshing_workflow/meshing_workflows.rst

```diff
@@ -572,6 +572,220 @@ Switch to solution mode
+            r"AddChild": r"yes",
+            r"BLControlName": r"aspect-ratio_1",
+            r"NumberOfLayers": 4,
+            r"OffsetMethodType": r"aspect-ratio",
+        }
+    )
+    meshing.workflow.TaskObject["Add 2D Boundary Layers"].AddChildAndUpdate(
+        DeferUpdate=False
+    )
+
+Generate the surface mesh
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Arguments.set_state(
+        {
+            r"Surface2DPreferences": {
+                r"MergeEdgeZonesBasedOnLabels": r"no",
+                r"MergeFaceZonesBasedOnLabels": r"no",
+                r"ShowAdvancedOptions": True,
+            },
+        }
+    )
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Execute()
+
+    meshing.workflow.TaskObject["aspect-ratio_1"].Revert()
+    meshing.workflow.TaskObject["aspect-ratio_1"].Arguments.set_state(
+        {
+            r"AddChild": r"yes",
+            r"BLControlName": r"uniform_1",
+            r"FirstLayerHeight": 2,
+            r"NumberOfLayers": 4,
+            r"OffsetMethodType": r"uniform",
+        }
+    )
+    meshing.workflow.TaskObject["aspect-ratio_1"].Execute()
+
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Arguments.set_state(None)
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Arguments.set_state(
+        {
+            r"Surface2DPreferences": {
+                r"MergeEdgeZonesBasedOnLabels": r"no",
+                r"MergeFaceZonesBasedOnLabels": r"no",
+                r"ShowAdvancedOptions": True,
+            },
+        }
+    )
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Execute()
+
+    meshing.workflow.TaskObject["uniform_1"].Revert()
+    meshing.workflow.TaskObject["uniform_1"].Arguments.set_state(
+        {
+            r"AddChild": r"yes",
+            r"BLControlName": r"smooth-transition_1",
+            r"FirstLayerHeight": 2,
+            r"NumberOfLayers": 7,
+            r"OffsetMethodType": r"smooth-transition",
+        }
+    )
+    meshing.workflow.TaskObject["uniform_1"].Execute()
+
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Arguments.set_state(None)
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Arguments.set_state(
+        {
+            r"Surface2DPreferences": {
+                r"MergeEdgeZonesBasedOnLabels": r"no",
+                r"MergeFaceZonesBasedOnLabels": r"no",
+                r"ShowAdvancedOptions": True,
+            },
+        }
+    )
+    meshing.workflow.TaskObject["Generate the Surface Mesh"].Execute()
+
+Export Fluent 2D mesh
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    meshing.workflow.TaskObject["Export Fluent 2D Mesh"].Arguments.set_state(
+        {
+            r"FileName": r"mesh1.msh.h5",
+        }
+    )
+    meshing.workflow.TaskObject["Export Fluent 2D Mesh"].Execute()
+
+Switch to solution mode
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Switching to solver is not allowed in 2D Meshing mode.
```

Comment:
```suggestion
Switching to solver mode is not allowed in 2D Meshing mode.
```
The title says "Switch to solution mode," yet the sentence says solver mode? Mistake maybe?